Makefile

```diff
@@ -196,6 +206,24 @@ OPM = $(shell which opm)
 endif
 endif
 
+.PHONY: operator-sdk
```

Comment:
Does the kuttl test need operator-sdk?

Reply:
kuttl-tests doesn't need operator-sdk but for bundling operator we would need it